linux/start/jupyter/adoc/gs_sles_jupyter-jupyterlab_container.adoc

```diff
@@ -463,7 +429,7 @@ including Alexis Cook's https://www.kaggle.com/code/alexisbcook/titanic-tutorial
 ----
 podman run --name jupyter \
  -p 8888:8888 \
- -v $(pwd):/home/jupyter/notebooks \
+ -v $(pwd):/home/jupyter/notebooks:Z \
```

Comment:
The doc describes running podman as a rootless user. That requires `--userns=keep-id` here, otherwise get `Permission denied: Untitled.ipynb` after clicking on notebook.

Actually with podman 3.4.7 `--userns=keep-id` only works if the host user id is 1000, which is the id of the `jupyter` user in the container.
https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md#using-volumes

Reply:
No, you only get a `permission denied` when the shared directory is not writeable by the `jupyter` user. Afaik the only way to work around that is to `chmod 0777 .` before invoking podman. At least `--userns=keep-id` has no influence on the result in my tests with podman 4.3.1